src/nl/hannahsten/texifyidea/psi/LatexPsiImplUtil.java

```diff
@@ -130,16 +142,16 @@ private static String stripGroup(String text) {
                 .filter(Objects::nonNull)
                 .map(PsiElement::getText)
                 .filter(Objects::nonNull)
+                // split the text elements along the comma separator
                 .flatMap(text -> OPTIONAL_SPLIT.splitAsStream(text))
+                .filter(text -> !text.isEmpty())
+                // return only the parameter name for parameters like "param=value"
```

Comment:
I'm not entirely clear on why you changed this. Is there any reason why we wouldn't also want the value?

For example, this will fail now:
```latex
\documentclass{article}
\usepackage[xindy]{imakeidx}
\makeindex[name=myindex]
\begin{document}
    Some random\index{random} text\index[myindex]{text} which should be indexed\index{index}.
    \printindex[myindex]
\end{document}
```
because we get optional parameters of the `\makeindex` command, and we do need to know the name of the index to copy.

Reply:
> Is there any reason why we wouldn't also want the value?

The reason why I changed it was because the javadoc states it should return only the names and from what I could tell no client made use of the parameter values (please correct me if I am wrong). It would be nice to have a method which also returns the values (e.g. as a Map), but this method would also have to take care of parameters like ```\makeindex[name={myindex}]``` which the previous implementation did not. But I can definitely extend the current method if needed.

> because we get optional parameters of the `\makeindex` command, and we do need to know the name of the index to copy.

Could you explain the example in more detail please? What do you mean with "know the name of the index to copy"?

Reply:
Ah, well I could only find one client which made use of it: the makeindex run configuration, which does the splitting on `=` itself to create a map of it (MakeindexRunConfiguration:162). Would probably be better to put that in a separate method indeed, and leave the one you changed as it now is.

In that example, imakeidx generates `myindex.idx`, on which we run texindy, and then copy the generated `myindex.ind` back next to the main file, because imakeidx is not so smart and can't find the .ind file otherwise. It's not so important how, but I believe we do need the value of the optional parameters of \makeindex there.

Reply:
> Ah, well I could only find one client which made use of it: the makeindex run configuration, which does the splitting on `=` itself to create a map of it (MakeindexRunConfiguration:162). Would probably be better to put that in a separate method indeed, and leave the one you fixed as it now is.

I must have missed it then. I will add a version that also returns the parameter values.

Reply:
@PHPirates I updated the implementation of getOptionalParameters now to create a map of parameters and values and also updated the existing clients to use the maps keyset where applicable.